prov/efa/src/rdm/rxr_pkt_type_base.c

```diff
@@ -108,7 +108,8 @@ int rxr_pkt_init_data_from_op_entry(struct rxr_ep *ep,
 	int tx_iov_index;
 	char *data;
 	size_t tx_iov_offset, copied;
-	struct efa_mr *desc;
+	bool iov_accessible_by_device;
```

Comment:
nit: I think it's better to initialize it to `false` here in case it falls through and remain undefined.

Reply:
set to `false` in new revision.